websites/T/TMBW/metadata.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "name": "Anant",
+    "id": "6629851641579044874"
+  },
+  "service": "TMBW",
+  "description": {
+    "en": "Displays what article you're reading on the They Might Be Giants fan-maintained wiki, TMBW.net."
```

Use the service's official description.
```suggestion
    "en": "This Might Be A Wiki is an ever-growing reference for all things They Might Be Giants."
```